### PR TITLE
Pr/17065

### PR DESCRIPTION
**Pull request by wutthichai46**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/formdata/using_formdata_objects/index.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ formElem.addEventListener('formdata', (e) => {
 
   // Get the form data from the event object
   let data = e.formData;
-  for (var value of data.values()) {
+  for (let value of data.values()) {
```
Comment (teoli2003) [inline files/en-us/web/api/formdata/using_formdata_objects/index.md:165]:
```suggestion
  for (const value of data.values()) {
```

File: files/en-us/web/api/formdataevent/formdata/index.md [hunk 2]
```diff
@@ -44,12 +44,12 @@ formElem.addEventListener('formdata', (e) => {
 
   // Get the form data from the event object
   let data = e.formData;
-  for (var value of data.values()) {
+  for (let value of data.values()) {
```
Comment (teoli2003) [inline files/en-us/web/api/formdataevent/formdata/index.md:47]:
```suggestion
  for (const value of data.values()) {
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thank you! Two tiny changes (`for...of` can use `const` unlike `for(;;)` that needs `let`).

**Comment by Josh-Cena** [comment]
This is a re-submission of #17065. @wutthichai46 judging from #16701 and #17956, I'd like to kindly ask you to adjust your behavior and do not re-submit other people's work.

**Pull request closed without merging**